src/julia-parser.scm

```diff
@@ -1577,8 +1577,13 @@
                       (else #t)))
          (rest  (if done
                     '()
-                    (parse-comma-separated s (lambda (s)
-                                               (parse-import s word))))))
+                    (let ((ex (parse-comma-separated s (lambda (s)
+                                                               (parse-import s word)))))
```

Comment:
Indentation

Reply:
What exactly is the rule for indentation here? I thought this is how lambdas should be aligned. Or should I make the whole lambda a separate line?

Reply:
See the previous version of the code.